test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -415,6 +416,47 @@ public void AddPicture_too_large_adds_svg_picture()
         pres.Validate();
     }
 
+    [Test]
+    [Explicit]
+    public void AddPicture_svg_with_text_matches_reference()
```

Comment:
What about such edits?

```C#
[Test]
public void AddPicture_svg_with_text_matches_reference()
{
    // ARRANGE

    // This presentation contains the same SVG we're adding below, manually
    // dragged in while running PowerPoint
    var pres = new Presentation(StreamOf("055_svg_with_text.pptx"));
    var shapes = pres.Slides[0].Shapes;
    var image = TestHelper.GetStream("1x1.svg");
    image.Position = 0;

    // ACT
    shapes.AddPicture(image);

    // ASSERT
    var picture = (IPicture)shapes.First(shape => shape.Name.StartsWith("Picture"));
    var xml = new XmlDocument { PreserveWhitespace = true };
    xml.LoadXml(picture.SvgContent);
    var textTagRandomChild = xml.GetElementsByTagName("text").OfType<XmlElement>().First().ChildNodes.Item(0);
    textTagRandomChild.Should().NotBeOfType<XmlSignificantWhitespace>("Text tags must not contain whitespace");
    
    // The above assertion does guard against the root cause of the bug 
    // which led to this test. However, the true test comes from loading
    // these up in PowerPoint and ensure the added image looks like the
    // existing image.
    pres.Validate();
}
```

Regard `xml.GetElementsByTagName("text").OfType<XmlElement>().First().ChildNodes.Item(0)`,

I think using statements as `for`, `if`, etc. in unit test make the test much more difficult to understand. If there is a case when, for example, `.ChildNodes.Item(0).Should().NotBeOfType<XmlSignificantWhitespace>()` is `true`, but `.ChildNodes.Item(1).Should().NotBeOfType<XmlSignificantWhitespace>()` is `false`, we'd better let such an error appear in production😁 and we'd better write a separate test for such case with a better understanding of what's going on.

Reply:
If this is how you'd prefer to do it, that's fine. The facts here are that text elements should not have ANY children which are whitespace. In this particular case, when it was broken, items 0 and 2 were whitespace. If something broke in the future where just item 1 was whitespace, that still would be broken.